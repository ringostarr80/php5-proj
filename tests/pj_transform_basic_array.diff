001+ The x-array passed contains 1 elements
002+ The y-array passed contains 1 elements
003+ max_count: 1
004+ float(-1)
001- array(3) {
002-   ["x"]=>
003-   float(0.25261488335704)
004-   ["y"]=>
005-   float(0.87388279965635)
006-   ["z"]=>
007-   float(0)
008- }