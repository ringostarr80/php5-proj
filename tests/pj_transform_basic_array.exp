array(3) {
  ["x"]=>
  float(0.25261488335704)
  ["y"]=>
  float(0.87388279965635)
  ["z"]=>
  float(0)
}