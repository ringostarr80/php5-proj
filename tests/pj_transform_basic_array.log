
---- EXPECTED OUTPUT
array(3) {
  ["x"]=>
  float(0.25261488335704)
  ["y"]=>
  float(0.87388279965635)
  ["z"]=>
  float(0)
}
---- ACTUAL OUTPUT
The x-array passed contains 1 elements
The y-array passed contains 1 elements
max_count: 1
float(-1)
---- FAILED
