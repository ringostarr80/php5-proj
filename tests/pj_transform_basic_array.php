<?php
$pj_krovak = pj_init_plus('+proj=krovak +lat_0=49.5 +lon_0=24.83333333333333 +alpha=0 +k=0.9999 +x_0=0 +y_0=0 +ellps=bessel');
if ($pj_krovak === false) {
	die(pj_strerrno(pj_get_errno_ref()));
}
$pj_latlong = pj_init_plus("+proj=latlong +towgs84=570.8,85.7,462.8,4.998,1.587,5.261,3.56 +units=m");
if ($pj_latlong === false) {
	die(pj_strerrno(pj_get_errno_ref()));
}
$x = array(-739443);
$y = array(-1045546);
$p = pj_transform($pj_krovak, $pj_latlong, 1, 0, $x, $y);
var_dump($p);
?>
