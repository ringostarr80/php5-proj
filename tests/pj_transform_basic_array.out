The x-array passed contains 1 elements
The y-array passed contains 1 elements
max_count: 1
float(-1)